R/importTaxpasta.R

```diff
@@ -0,0 +1,116 @@
+#' Import taxpasta-specific BIOM results to `TreeSummarizedExperiment`
+#'
+#' @param filename A character vector with one element that denotes the file
+#'   path to a BIOM file.
+#'
+#' @return A [TreeSummarizedExperiment::TreeSummarizedExperiment()] object.
+#' @export
+#'
+#' @examples
+#' tse <- importTaxpasta("profiles.biom")
+#'
+#' @importFrom rhdf5 h5read
+#' @importFrom SummarizedExperiment rowData
+#' @importFrom SingleCellExperiment altExps
```

Comment:
Add NULL here as a last line after documentation. Then export functions "manually" by adding @export before the function description (check other functions/files).

Now there are also manual pages for internal functions which we do not want. 

Also, direct this function's documentation to same file where all importers are located. Check for example documentation of importBIOM

Reply:
The only function that I want to export is `importTaxpasta` and it has the `@export` statement.

> Now there are also manual pages for internal functions which we do not want.

Right, I will exclude them.

> Also, direct this function's documentation to same file where all importers are located. Check for example documentation of importBIOM

I'll look into that.

Reply:
Here is an example: https://github.com/EBI-Metagenomics/HoloFoodR/blob/devel/R/getMetaboLights.R

By using NULL and @export just above the function, you can specify which functions are exported and listed in the documentation.

Reply:
It seems to me the same can be achieved by specifying on the helper functions.

```r
#' @noRd
```